### PR TITLE
#90 fix not worked enableBodyScroll in some conditions

### PR DESCRIPTION
**Pull request by An-Cu**
here is commit to fix issue #90

### CODE REVIEW

File: src/bodyScrollLock.js [hunk 1]
```diff
@@ -230,10 +230,12 @@ export const enableBodyScroll = (targetElement: any): void => {
...
   } else if (locks.length === 1 && locks[0].targetElement === targetElement) {
-    restoreOverflowSetting();
-
     locks = [];
   } else {
     locks = locks.filter(lock => lock.targetElement !== targetElement);
   }
+
+  if (!locks.length) {
+    restoreOverflowSetting();
```
Comment (willmcpo) [inline src/bodyScrollLock.js]: why do we need to run this for even ios devices?

Comment (An-Cu) [reply src/bodyScrollLock.js]: @willcpo I don't tested changes on iOS.
Anyway the same issue can appear on any device, so I assume that this change is required for iOS devices too.

**Comment by An-Cu** [comment]
I can change condition and check if overflow hidden was set and only then invoke  restoreOverflowSetting. But moving              restoreOverflowSetting to condition above is bad idea, because then code will be more complicated and unreadable through adding same piece of code inside `else` and `else if` blocks.( WET vs. DRY principle)

**Comment by willmcpo** [comment]
Refer to my comment about removing else if block

**Comment by An-Cu** [comment]
Ok it seems good for me.
Sorry for misunderstanding.

I will add commit on Wednesday.

**Comment by An-Cu** [comment]
@willmcpo I updated pull-request.

**Comment by willmcpo** [comment]
thanks for the fix @An-Cu

**Pull request merged**